docs/lazy-adr/adr-005-decouple-blockid-and-partsetheader.md

```diff
@@ -0,0 +1,49 @@
+# ADR 005: Decouple the 'PartSetHeader' from the 'BlockID'
+
+## Changelog
+
+- 2021-08-01: Initial Draft
+
+## Context
+
+Celestia has multiple commits to the block data via the 'DataHash' and the 'PartSetHeader' in the 'BlockID'. As stated in the [#184](https://github.com/celestiaorg/lazyledger-core/issues/184), we no longer need the 'PartSetHeader' for this additional commitment to the block’s data. However, in the short-medium term, we are still planning on using the 'PartSetHeader' to be used for block propagation during consensus. This means that we will remove the 'PartSetHeader' from as many places as possible, but keep it in the 'Proposal' struct.
+
+## Alternative Approaches
+
+It’s worth noting that there are proposed changes to remove the 'PartSetHeader' entirely, and instead use the already existing commitment to block data, the 'DataAvailabilityHeader', to propagate blocks in parallel during consensus. Discussions regarding the detailed differences entailed in each approach are documented in that ADR’s PR. The current direction that is described in this ADR is significantly more conservative in its approach, but it is not strictly an alternative to other designs. This is because other designs would also require removal of the 'PartSethHeader', which is a project in and of itself due to the 'BlockID' widespread usage throughout tendermint and the bugs that pop up when attempting to remove it. 
```

Comment:
```suggestion
It’s worth noting that there are proposed changes to remove the `PartSetHeader` entirely, and instead use the already existing commitment to block data, the `DataAvailabilityHeader`, to propagate blocks in parallel during consensus. Discussions regarding the detailed differences entailed in each approach are documented in that ADR’s PR. The current direction that is described in this ADR is significantly more conservative in its approach, but it is not strictly an alternative to other designs. This is because other designs would also require removal of the `PartSethHeader`, which is a project in and of itself due to the `BlockID` widespread usage throughout tendermint and the bugs that pop up when attempting to remove it. 
```